### PR TITLE
Create resource with ID and fix search count

### PR DESCRIPTION
**Pull request by simonvadee**
- fix search (translate _count to integer for elastic)
- add nginx config to allow CORS
- allow creating resource with id

---
This change is [Reviewable](https://reviewable.io/reviews/arkhn/warehouse-api/32)

### CODE REVIEW

File: fhir-api/api.py [hunk 1]
```diff
@@ -94,10 +94,10 @@ def search(resource_type):
     offset = 0
     total = 100
 
-    if parsed_params["_count"]:
+    if parsed_params.get("_count"):
```
Comment (elsiehoffet-94) [inline fhir-api/api.py]: Why do you prefer this?

File: docker-compose.yml [hunk 2]
```diff
@@ -9,6 +9,8 @@ services:
       - 80:80
     depends_on:
       - fhir
+    networks:
```
Comment (elsiehoffet-94) [inline docker-compose.yml]: Out of curiosity, what does this do ?

File: fhir-api/api.py (hunk 1 above)
Comment (Jasopaum) [reply fhir-api/api.py]: It was my mistake, if "_count" is not in parsed_params, `parsed_params["_count"]` will raise KeyError

File: docker-compose.yml (hunk 2 above)
Comment (simonvadee) [reply docker-compose.yml]: it puts all the docker services in the same network, that way they can talk to each other even if you do not specify all the docker-compose files in the command.
eg:
```
docker-compose -f service_A.yml up
docker-compose -f service_B.yml up
# service A et B can talk to each other
```

File: fhir-api/api.py (hunk 1 above)
Comment (simonvadee) [reply fhir-api/api.py]: if "_count" is not present, it raises a KeyError (it does not return None)

File: docker-compose.yml (hunk 2 above)
Comment (elsiehoffet-94) [reply docker-compose.yml]: Awesome !

**Pull request merged**